This patch is created by Oracle to:

  1. Remove reference in named(8) to lwresd(8) which is not, and has
     never been, provide with Solaris distribution.

  2. Add reference to dns-server(8s), SMF manual page.

This patch is Solaris specific and is not applicable to upstream.
diff --git a/bin/named/named.8 b/bin/named/named.8
index cd990a9..7c4b897 100644
--- a/bin/named/named.8
+++ b/bin/named/named.8
@@ -367,9 +367,24 @@ RFC 1035,
 \fBnamed-checkconf\fR(8),
 \fBnamed-checkzone\fR(8),
 \fBrndc\fR(8),
-\fBlwresd\fR(8),
+\fBdns-server\fR(8s),
 \fBnamed.conf\fR(5),
 BIND 9 Administrator Reference Manual\&.
+.PP
+The BIND 9 \fIAdministrator's Reference Manual\fR is available
+online or, if locally installed, at
+file:///usr/share/doc/bind/html/Bv9ARM.html and
+file:///usr/share/doc/bind/Bv9ARM.pdf.
+.PP
+.br
+.ps +1
+\fBNote\fR
+.ps -1
+.br
+Some compile-time features documented in the ARM are not available; to
+see a list of enabled features run '\fB/usr/sbin/named -V\fR'.
+.sp .5v
+.PP
 .SH "AUTHOR"
 .PP
 \fBInternet Systems Consortium, Inc\&.\fR
